src/functions-and-arrays.js

```diff
@@ -1,73 +1,450 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (num1, num2) => {
+  if (num1 > num2) return num1;
+  else if (num2 > num1) return num2;
+  else return num1;
+};
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+const findLongestWord = words => {
+  let longestWord = "";
+  if (words.length === 0) return null;
+  else {
+    for (let i = 0; i < words.length; i++) {
+      if (words[i].length > longestWord.length) {
+        longestWord = words[i];
+      }
+    }
+    return longestWord;
+  }
+};
 
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+const sumNumbers = array => array.reduce((acc, curval) => acc + curval, 0);
```

Comment:
Nice!

Also, if you dont set any default values on reduce it will automatically start at 0, so there is no need to set the default value in this particular case